processor/src/bitwise/utils.rs

```diff
@@ -0,0 +1,10 @@
+use super::{ExecutionError, Felt, StarkField};
+
+pub fn assert_u32(value: Felt) -> Result<Felt, ExecutionError> {
+    let val_u64 = value.as_int();
+    if val_u64 > u32::MAX.into() {
+        Err(ExecutionError::NotU32Value(value))
+    } else {
+        Ok(value)
+    }
+}
```

Comment:
Let's move this function into `mod.rs`. You can add a "HELPER FUNCTIONS" section at the end of the file and put this function there.